mmv1/products/looker/Instance.yaml

```
@@ -0,0 +1,368 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Instance'
+base_url: projects/{{project}}/locations/{{region}}/instances
+create_url: projects/{{project}}/locations/{{region}}/instances?instanceId={{name}}
+update_verb: :PATCH
+update_mask: true
+description: |
+  A Google Cloud Looker instance.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Create a Looker (Google Cloud core) instance': 'https://cloud.google.com/looker/docs/looker-core-instance-create'
+    'Configure a Looker (Google Cloud core) instance': 'https://cloud.google.com/looker/docs/looker-core-instance-setup'
+  api: 'https://cloud.google.com/looker/docs/reference/rest/v1/projects.locations.instances'
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 90
+  update_minutes: 90
+  delete_minutes: 60
+autogen_async: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'looker_instance_basic'
+    primary_resource_id: 'looker-instance'
+    vars:
+      instance_name: 'my-instance'
+      client_id: 'my-client-id'
+      client_secret: 'my-client-secret'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'looker_instance_full'
+    primary_resource_id: 'looker-instance'
+    vars:
+      instance_name: 'my-instance'
+      client_id: 'my-client-id'
+      client_secret: 'my-client-secret'
+parameters:
+  # TODO: resourceref?
+  - !ruby/object:Api::Type::String
+    name: 'region'
+    description: |
+      The name of the Looker region of the instance.
+    required: false
+    immutable: true
+    url_param_only: true
+    ignore_read: true
+    default_from_api: true
```

```suggestion
    immutable: true
    url_param_only: true
    default_from_api: true
```

ignore_read is redundant with url_param_only, `required: false` is redundant as it defaults to false.